language/move-prover/bytecode-to-boogie/src/spec_translator.rs

```diff
@@ -56,7 +56,7 @@ impl<'a> SpecTranslator<'a> {
     /// Reports an error. At the moment, we simply abort.
     // TODO: extend this to collect errors and add source info. Code needs to expect this
     //   returns with the given `pass_through` value.
-    pub fn error<T>(&mut self, msg: &str, _pass_through: T) -> T {
+    pub fn error<T>(&self, msg: &str, _pass_through: T) -> T {
```

Comment:
This will eventually need to be mutual, as the errors will be collected instead of bailing out. Let's keep it that way so code emitting `error` stays correct.